runtime_scan/pkg/orchestrator/common/reconciler.go

```diff
@@ -52,6 +52,7 @@ func (r *Reconciler[T]) Start(ctx context.Context) {
 				}
 				cancel()
 			}
+			r.Queue.Done(item)
```

Comment:
@sambetts-cisco If `Dequeue` returned an error, should we call `Done` here?

Reply:
I actually fixed that in the next PR, I think this is a bug... item will be an empty T so it won't break because of the implementation but its not good either. I'll push a revision thanks for catching!